content/app/app-dev-course/_index.en.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Intro course
-description: An introductory course to app development with Altinn Studio and local dev tools. (Only available in Norwegian)
```

Comment:
Ta en ny titt på denne siden. Nederst. Ser ut som listen som er der ikke er formatert korrekt

Reply:
Gjelder både på norsk og engelsk.

Reply:
Linje 15.
-  Lenken der peker på norsk side. 
- Den engelske siden man sendes til har enda "translate-to-english" tag på seg. Denne kan fjernes